src/calculateRentalCost.js

```
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 3) {
+    return 100;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 7) {
+    return 230;
+  }
+
+  return 80;
 }
```

This implementation doesn't follow the logic described in the task. Instead of calculating the cost based on the number of days and applying discounts, it returns hardcoded values for specific inputs. This causes it to fail for many cases, for example, `calculateRentalCost(1)` returns `80` instead of the expected `40`.

Please refactor the function to:
1.  Calculate the base cost (`days * daily_rate`).
2.  Apply the correct discount based on the number of days (`>= 7` or `>= 3`).

Additionally, this violates checklist item #2: 'Don't use literals in expressions (HARDCODE)'. Remember to define constants for values like the daily rate, discount thresholds, and discount amounts.